spec/ruby/core/gc/stat_spec.rb

```diff
@@ -0,0 +1,21 @@
+require_relative '../../spec_helper'
+
+describe "GC.stat" do
+  it "returns zero for unknown key" do
+    keys = [:omganyvalue, :total_allocated_objects]
+    keys.each do |key|
+      GC.stat(key).should == 0
```

Comment:
We should just check `should be_kind_of(Integer)` here, as it won't be 0 on MRI.